extension/test/unit/lib/magento/customer/addItems-test.js

```diff
@@ -0,0 +1,52 @@
+const assert = require('assert')
+const request = require('request-promise-native')
+const nock = require('nock')
+
+const getWishlistId = require('../../../../../lib/magento/customer/getWishlistId')
+const transformItemId = require('../../../../../lib/transformItemId')
+
+describe('magento/customer: addItems step', () => {
+  let input = {}
+  let context = {}
+  const magentoUrl = 'http://localhost/shopgate/v2'
+  const path = '/wishlists'
+
+  beforeEach(() => {
+    input = {
+      token: 'testToken'
+    }
+
+    context = {
+      meta: {
+        userId: 'shopgate@test.com'
+      },
+      config: {
+        magentoUrl,
+        allowSelfSignedCertificate: true
+      },
+      tracedRequest: () => {
+        return request
+      },
+      log: {
+        debug: (object, message) => {
+        }
+      }
+    }
+  })
+  it('Test that getWishlist returns the first exist wischlistId', async () => {
+    nock(magentoUrl).get(path).reply(200, [{ wishlist_id: 2 }, { wishlist_id: 1 }])
```

Comment:
Would be better if you start the name with the expectation. e. g. something like  it('should return the first existing wischlistId'